reset-database.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+while :; do
+  echo -n "Provide openbox db name: "
+  read DATABASE_NAME
+
+  if mysql -Nse 'SHOW DATABASES;' | grep "${DATABASE_NAME}" -q; then
+    break
+  else
+    echo "Database ${DATABASE_NAME} does not exist. Try again"
+  fi
+done
+
+# User confirmation
+echo "Are you sure you want to delete data from ${DATABASE_NAME}?"
```

Comment:
We should probably describe the full extent to which we will be destroying their data. And then ask them if they are sure.

Reply:
I added line, that database will be deleted, not sure if it's enough